### ISSUE TITLE
Am currently unable to compile this in BD 

### ISSUE THREAD

**Issue by lilyspad** [M0nsterdev/freemoji#2]
As stated in title, is this plugin still being developed?

**Comment by M0nsterdev** [comment]
No, this was dropped a few months ago, when discord was updated, the bd team has their own that they have on their website
